readme/Configuration.scalatex

```diff
@@ -381,6 +381,26 @@
       implicit def validatedInstances[E](implicit E: Semigroup[E]): Traverse[
           Validated[E, ?]] with ApplicativeError[Validated[E, ?], E] = 2
 
+  @sect{newlines.beforeCaseClassParameterDefn}
+    Together with @b{verticalMultilineAtDefinitionSite}, this will
+    align parameter definitions vertically, with a line for each, and
+    not attempt to fit the case class definition on a single line.
```

Comment:
It might be worth to mention that this might be desirable for cleaner git merges. Can we also mention this option under [this section](http://scalameta.org/scalafmt/#Whichconfigurationoptionsminimizediffs/conflictsinversioncontrol?) since it might be of interest to other people.